src/screens/Services/Services.js

```diff
@@ -213,7 +213,22 @@ class ServicesScreen extends React.Component<Props, State> {
           const { user: { email } } = this.props;
           const address = this.getCryptoPurchaseAddress();
           if (address === null) return;
-          this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email));
+
+          Alert.alert(
+            t('servicesContent.ramp.assetDecisionAlert.title'),
+            t('servicesContent.ramp.assetDecisionAlert.description'),
+            [
+              {
+                text: t('servicesContent.ramp.assetDecisionAlert.actionNonPlr'),
+                onPress: () => this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email, false)),
```

Comment:
nit: you can just ommit the `false` because `plrMode` in `rampWidgetUrl ` is optional

Reply:
Great spot! This has been corrected.